src/helics/application_api/ValueFederate.cpp

```diff
@@ -153,12 +154,25 @@ static const std::string emptyStr;
 template <class Inp, class Obj>
 static void loadOptions (ValueFederate *fed, const Inp &data, Obj &objUpdate)
 {
-    // bool optional = getOrDefault (data, "optional", false);
+    addTargets (data, "flags", [&objUpdate](const std::string &target) {
```

Comment:
@phlptp I was expecting a function like this for MessageFederate.cpp for configuring flag options. Also I noticed that this function does not support all the possible flags listed in helics_handle_options enumeration. I understand if it doesn't make sense for some of the flags to apply to an Input or a Publication but I thought the strict type checking flag should be added as well.

Reply:
I think all the flags are available in getOptionIndex now so can be set through the flags field

Reply:
excellent!